x-pack/test/functional/apps/monitoring/index.js

```diff
@@ -46,7 +46,7 @@ export default function ({ loadTestFile }) {
     loadTestFile(require.resolve('./beats/listing'));
     loadTestFile(require.resolve('./beats/beat_detail'));
 
-    loadTestFile(require.resolve('./time_filter'));
+    // loadTestFile(require.resolve('./time_filter'));
```

Comment:
Tests inside this file are skipped, so I commented the whole line to avoid the alert modal breaking the code in the `before` method.